README.md

```diff
@@ -47,23 +47,25 @@ source /path-to-where-spack-was-cloned/spack/share/spack/setup-env.sh
 Currently, most of the dependencies are listed in `spack.yaml` and can be installed via Spack. `gcc` and `g++` will be
 needed to build ChronoLog.
 
-A Spack environment needs to be created and activated using the following commands. When the environment is activated, a
-shell prompt `[ChronoLog]` will pop up.
+A Spack environment needs to be created and activated using the following commands. 
 
 ```
 cd ChronoLog
 git switch develop
 spack env activate -p .
 spack install -v
```

Comment:
move `spack install -v` to be after checking spack env section. make it an one-time step to install all dependencies in spack so that following build of ChronoLog does not require an installation step.